docker-compose.yaml

```diff
@@ -2,10 +2,12 @@ version: "3.3"
 services:
   waterfall:
     container_name: waterfall
-    image: ghcr.io/matrix-org/waterfall:main
+    # Use a proper commit hash for specific deployments.
+    image: ghcr.io/matrix-org/waterfall:sha-b64edee
```

Comment:
Currently we only need to update it on the development machine that hosts the SFU whenever we want to re-deploy (e.g. instead of `scp`ing a locally built executable [that must be built on a compatible Linux machine]), we just replace the hash with the has of the commit that we want to have and simply `docker-compose up` it.

But once https://github.com/matrix-org/waterfall/pull/49 is done, we don't need to do manual action as I would replace the `:tag` potion with something like `:${GIT_COMMIT_IT}` which then would be automatically filled up by a CD pipeline. I'm currently not sure how we're going to do this via GitHub (hence the open https://github.com/matrix-org/waterfall/pull/49), but I know how to automate it completely with e.g. GitLab.

Reply:
Ah ok, fair enough if it's only temporary.